foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -27,12 +25,13 @@ def source_type(self):
         return "ARRAY_EXPRESS"
 
     def downloader_task(self):
-        return download_array_express
+        return ("data_refinery_workers.downloaders"
+                + ".array_express.download_array_express")
```

Comment:
Yeah - I think it is better to define it as a constant up top. That way we know it is currently set at whatever value it is. I think it would not be too far-fetched for the module name to change at some point.